src/libs/actions/Report.js

```diff
@@ -385,20 +387,23 @@ function openReport(reportID, participantList = [], newReportObject = {}) {
             isOptimisticReport: true,
         };
 
-        const optimisticCreatedAction = ReportUtils.buildOptimisticCreatedReportAction(newReportObject.ownerEmail);
-        onyxData.optimisticData.push({
-            onyxMethod: Onyx.METHOD.SET,
-            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
-            value: {[optimisticCreatedAction.reportActionID]: optimisticCreatedAction},
-        });
-        onyxData.successData.push({
-            onyxMethod: Onyx.METHOD.MERGE,
-            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
-            value: {[optimisticCreatedAction.reportActionID]: {pendingAction: null}},
-        });
-
-        // Add the createdReportActionID parameter to the API call
-        params.createdReportActionID = optimisticCreatedAction.reportActionID;
+        // Add a created action, unless we are creating a thread
+        if (parentReportActionID === '0') {
```

Comment:
@stitesExpensify Can you please explain why we don't create the CREATED report action for threads? A thread is also a new report. We think this is causing a bug https://github.com/Expensify/App/issues/19490.

Reply:
I think that you're right and we should always do this. It looks like the back end is creating a CREATED action, so we should do it optimistically too.

Reply:
Thank you!